internal/db/migrations/003_accounts.up.sql

```diff
@@ -0,0 +1,10 @@
+BEGIN;
+
+CREATE TABLE accounts
+(
+    id BIGINT GENERATED BY DEFAULT AS IDENTITY PRIMARY KEY,
+    account_number TEXT NOT NULL UNIQUE CHECK (NOT empty(account_number)),
```

Comment:
We don't know for sure, if I have to choose between get that info and store it to only drop it later or ignore this and solve when we encounter problems, I choose one. There is a transition period planned and some services will only accept the account number for some time, although there is the translation service why not to keep those numbers in our database since it will come for free in requests anyway? Dropping a column is literally a DDL one-liner.

Reply:
I am not against keeping the account_number, I want to allow a blank value for it. There are already some CUs that will not have `account_number` and such CUs would not be able to use our service :)